scripts/build/termux_create_debian_subpackages.sh

```diff
@@ -101,11 +101,13 @@ termux_create_debian_subpackages() {
 		local PKG_DEPS_SPC=" ${TERMUX_PKG_DEPENDS//,/} "
 
 		if [ -z "$TERMUX_SUBPKG_DEPEND_ON_PARENT" ] && [ "${PKG_DEPS_SPC/ $SUB_PKG_NAME /}" = "$PKG_DEPS_SPC" ]; then
-		    TERMUX_SUBPKG_DEPENDS+=", $TERMUX_PKG_NAME (= $TERMUX_PKG_FULLVERSION)"
+			TERMUX_SUBPKG_DEPENDS+=", $TERMUX_PKG_NAME (= $TERMUX_PKG_FULLVERSION)"
+		elif [ "$TERMUX_SUBPKG_DEPEND_ON_PARENT" = force ]; then
+			TERMUX_SUBPKG_DEPENDS+=", $TERMUX_PKG_NAME (= $TERMUX_PKG_FULLVERSION)"
```

Comment:
I think I will go ahead with the force option. Because of the way each architecture main package have different DEPENDS list. The current build system will cause something like this to happen:

build arch | rust-std-aarch64* | rust-std-arm* | rust-std-i686* | rust-std-x86_64*
--- | --- | --- | --- | ---
aarch64 | not depend | depend | depend | depend
arm | depend | not depend | depend | depend
i686 | depend | depend | not depend | depend
x86_64 | depend | depend | depend | not depend

This is problematic if I want to make all the subpackages with neutral arch to be shared across arch and also locked to a particular version... I think the CI just overwrites the same file name deb then upload to server which 1/4 chance to get right...

Reply:
Or not, I guess I can drop this and go ahead with TERMUX_SUBPKG_DEPEND_ON_PARENT=no